### DISCUSSION TITLE
✅ Replace select control with choose control for Icon Position control in Button widget

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#22638] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi team.

Currently, in the button widget, the icon position option is select control. But as you know, choose controls are much better. Because they are both visually and easily selectable. So it would be great if you replace it with the current control.

### Describe the solution you'd like

please Replace select control with choose control for Icon Position control in Button widget. If in the future the possibility of setting the position of the icon at the top and bottom of the button text is also added, it will be great in the choose control (compared to select control).

![btn](https://github.com/elementor/elementor/assets/77163248/a10b5593-89e2-4157-929b-192ce71ac852)

**Comment by rami-elementor** [comment]
Hi @79ho3ein 

This request was fixed in Elementor 3.20.0, thank you for reporting.

**Reply by rami-elementor** [reply]
Ref: https://github.com/elementor/elementor/pull/25092

**Reply by 79ho3ein** [reply]
Thank you @rami-elementor🙏
